openshift/docs/webhook-setup.md

```diff
@@ -0,0 +1,22 @@
+## Deploying code changes via Github Webhooks in Openshift
+
+If you created an application and Cloudant instance from the Openshift Service Catalog, you might be wondering how you can can start iterating on your application. All you need to set up Continuous Delivery with Openshift is a Github Webhook that will tell Openshift to automatically pull in your newest changes to build and deploy.

```

Comment:
If you created an application from a template in the Openshift Container Platform Service Catalog and deployed it to your OpenShift cluster,

Reply:
done